apps/i18n/applab/en_us.json

```diff
@@ -486,7 +486,7 @@
   "makerSetupGeneralDescription": "In the Code.org [standalone Maker units](https://code.org/maker) and the Physical Computing unit in [Computer Science Discoveries](https://code.org/educate/csd) students use the [Circuit Playground or micro:bit](https://code.org/maker#explore_devices_link) connected to their computers via USB to allow for online curriculum integration and interactive debugging of programs via Maker Toolkit.\n\nMaker Toolkit is an add-on on top of [App Lab](https://code.org/educate/applab) our block-to-text app development environment and provides a special set of commands that allow student apps to communicate with external hardware.\n\nPlease follow the instructions below to connect your board to Code.org.",
   "makerSetupGeneralTitle": "Welcome Makers!",
   "makerSetupInstallSerialConnector": "Please install the [Code.org Serial Connector Chrome App]({webstoreURL}).",
-  "makerSetupInstallFirmataCP": "You should make sure it has the right firmware sketch installed. You can install the Firmata sketch [for Circuit Playground Express boards with these instructions]({firmataURLExpress}) and [for Circuit Playground Classic boards with these instructions]({firmataURLClassic}).",
+  "makerSetupInstallFirmataCP": "You should make sure it has the correct Firmata application installed. You can install the Firmata application for Circuit Playground Express boards by following the [instructions below](#{installInstructionsId})  and [for Circuit Playground Classic boards with these instructions]({firmataURLClassic}).",
```

Comment:
We have since added the CPX install instructions on the '/maker/setup' page below the checklist.

Reply:
For consistency with the first link, should we move the link text to be just around "these instructions"? 

`and [for Circuit Playground Classic boards with these instructions]` -> `and for Circuit Playground Classic boards with [these instructions]`